synthetic_data_generation.py

```diff
@@ -111,12 +113,41 @@ def noise_generator(x):
         axs[i].axvline(x=true_freq, color='k', linestyle='--', alpha=0.4) #plotting vertical lines at the true frequencies            
 
 fig.tight_layout()
+
+
 # %%
 # now that we better understand how to use frequency estimation tools
 
 # lets try to simulate a signal with powerline noise.
 
 # create a signal that has powerline noise at 60Hz and at 6 harmonics (120, 180, 240, ...)
 # plot 1: plot the signal
+t, sinewave = sine_wave_generator(DURATION_S, SAMPLE_RATE, FREQUENCY, n_harmonics=N_HAR) 
+FREQUENCY = 60
+SAMPLE_RATE = 2000
+DURATION_S = 10
+N_HAR = 6
+
+fig, (ax1, ax2) = plt.subplots(2, 1, figsize=(10,4), dpi=200)
+
+ax1.plot(t, sinewave, 'k')
+ax1.spines['top'].set_visible(False)
+ax1.spines['right'].set_visible(False)
+ax1.set_xlim([0,1])
+ax1.set_ylabel("Power")
+ax2.set_xlabel("Time (s)")
```

Comment:
Which axis will this set the `xlabel` for?

Reply:
Shoot I forgot to change that one